classes/class3/catscatscats/routes/mg_funcs.js

```diff
@@ -0,0 +1,121 @@
+var mongoose = require('mongoose');
```

Comment:
In the future you'll probably want an organizational scheme better than putting all your helper functions in the same file.